components/raftstore-v2/src/raft/storage.rs

```diff
@@ -49,18 +54,22 @@ pub fn write_initial_states(wb: &mut impl RaftLogBatch, region: Region) -> Resul
 /// A storage for raft.
 ///
 /// It's similar to `PeerStorage` in v1.
-pub struct Storage<ER> {
-    entry_storage: EntryStorage<ER>,
+pub struct Storage<EK: KvEngine, ER> {
```

Comment:
Is `:KvEngine` necessary?

Reply:
yes, otherwise it reports as unsafe to send.